migaloo/assetlist.json

```diff
@@ -83,6 +83,41 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/fable.svg"
       }
-    }
+    },
+      {
+        "description": "boneLUNA are autocompounding LUNA tokens",
+        "denom_units": [
+          {
+            "denom": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+            "exponent": 0
+          },
+          {
+            "denom": "bluna",
+            "exponent": 6
+          }
+        ],
+        "base": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+	    "type_asset": "ics20",
+        "name": "boneLuna",
+        "display": "bluna",
+        "symbol": "bLUNA",
+        "traces": [
+          {
+            "type": "ibc",
```

Comment:
since it's transferring a cw20 token, this should actually be type 'ibc-cw20', requiring 'port' property for both directions (I should have provided a more appropriate example)

Reply:
Here's an example of a cw20 ibc trace: https://github.com/osmosis-labs/assetlists/blob/d7ae22c9a7222974c58fd2f54ce1e50d42cb1bfc/osmosis-1/osmosis-1.assetlist.json#L2361-L2376